k8s/mlflow/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
+{{/*{{- if false }}*/}}
```

Comment:
Hi @Mnoble-19 , what is the significance of this file? Part of the config files? Or just to temporarily store some information?

Reply:
Once the application is deployed, information about the node port and ip to access the application is printed in the terminal, using the contents of this file as a template, thus me committing it as well